I do magic/promises-callbacks-js/src/js/load-data.js

```diff
@@ -0,0 +1,66 @@
+function validateEnter(e) {
+  if (e.keyCode===13) getAgents('https://api.myjson.com/bins/uptto');
+}
+
+function get(url) {
+  // Return a new promise.
+  return new Promise(function(resolve, reject) {
```

Comment:
Use arrow functions instead
return new Promise((resolve, reject) => { ... })

https://medium.freecodecamp.org/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26

Reply:
Done